pyvista/plotting/charts.py

```diff
@@ -1376,20 +1373,15 @@ def show(
         ----------
         off_screen : bool, optional
             Plots off screen when ``True``.  Helpful for saving screenshots
-            without a window popping up.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            without a window popping up. Defaults to active theme setting.
 
         full_screen : bool, optional
             Opens window in full screen.  When enabled, ignores
-            ``window_size``.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            ``window_size``.
```

Comment:
```suggestion
            ``window_size``. Defaults to active theme setting.
```
Keep this in the description now that we reverted to the 'optional' keyword. Linking is also gone now, but maybe not necessary? Just make it consistent with `window_size` (which still has linking now) and `off_screen` (which doesn't).